src/lib/zkapp.ts

```diff
@@ -661,7 +662,7 @@ class SmartContract {
    * it so that proofs end up in the original finite field). These are fairly expensive operations, so **expect compiling to take at least 20 seconds**,
    * up to several minutes if your circuit is large or your hardware is not optimal for these operations.
    */
-  static async compile() {
+  static async compile({ cache = Cache.FileSystemDefault } = {}) {
```

Comment:
`SmartContract.compile()` now expects an optional `Cache`. same for `ZkProgram.compile()`